backend/src/charityprojects/api/views.py

```diff
@@ -230,3 +230,31 @@ def update_project_challenge_status_ideation(request):
             return Response(status=status.HTTP_201_CREATED)
         else:
             return Response(status=status.HTTP_400_BAD_REQUEST)
+
+
+def update_user_invitation(request):
+    response = {'status': "Invalid Request"}
+    if request.method == 'POST':
+        invited_user_list = []
```

Comment:
Not used anywhere only values inserted into this list but not made use of. Is it needed for future work?